src/System.Globalization.Calendars/tests/CalendarTestWithConfigSwitch/Configurations.props

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp;
```

Comment:
>netcoreapp [](start = 6, length = 10)

and this test is only for netcoreapp. does the intend to have the tests are same in different branches?